lib/network/http2_network_test.go

```diff
@@ -8,33 +8,29 @@ import (
 	"testing"
 	"unicode"
 
-	"boscoin.io/sebak/lib/common"
-	"github.com/stellar/go/keypair"
-	"github.com/stretchr/testify/assert"
 	"math/rand"
 	"net"
 	"strconv"
-	"sync"
 	"time"
-)
 
-var testPort = "5000"
-var once sync.Once
+	"boscoin.io/sebak/lib/common"
+	"github.com/stellar/go/keypair"
+	"github.com/stretchr/testify/assert"
+)
 
 func getPort() string {
-	once.Do(func() {
-		for {
-			s := rand.NewSource(int64(time.Now().Nanosecond()))
-			r := rand.New(s)
-			testPort = strconv.Itoa(r.Intn(16383) + 49152) // ephemeral ports range 49152 ~ 65535
-
-			ln, err := net.Listen("tcp", ":"+testPort)
-			if err == nil {
-				ln.Close()
-				break
-			}
+	var testPort = "5000"
+	for {
+		s := rand.NewSource(int64(time.Now().Nanosecond()))
+		r := rand.New(s)
+		testPort = strconv.Itoa(r.Intn(16383) + 49152) // ephemeral ports range 49152 ~ 65535
```

Comment:
For clarity, I'd write something like `r.Intn(65535 - EPHEMERAL_START) + EPHEMERAL_START`